test/testsimplifytemplate.cpp

```diff
@@ -5297,7 +5297,7 @@ class TestSimplifyTemplate : public TestFixture {
         ASSERT_EQUALS(expected, tok(code));
     }
 
-    unsigned int instantiateMatch(const char code[], const std::size_t numberOfArguments, const char patternAfter[]) {
+    bool instantiateMatch(const char code[], const std::size_t numberOfArguments, const char patternAfter[]) {
```

Comment:
this was doing an implicit and unnecessary bool->int conversion.

Reply:
good! this return type should match `TemplateSimplifier::instantiateMatch` return type.